serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala

```diff
@@ -18,52 +20,95 @@ class SerializabilityCheckerCompilerPluginComponent(
     new StdPhase(prev) {
       private val serializabilityTraitType = typeOf[SerializabilityTrait]
 
+      private val genericsNamesWithTypes = Map(
```

Comment:
Nit: either use `names` in the next two vals, or skip in all three for consistency (same for `genericsNames` & co.)